src/components/pages/python.tsx

```diff
@@ -0,0 +1,45 @@
+import '../../styles/python.scss';
+import BooleanDropdown from '../shared/booleanDropdown';
+
+function Python(): JSX.Element {
+
+  const outputAndMatches = {
+    num_of_antennas: '2',
+    has_arms: 'True',
+    has_ears: 'False',
+    color: '"white"',
+  };
+
+  return (
+    <div id='python'>
+      <h1>Match the output to the code:</h1>
+      <div id='problem'>
+        <div id='code'>
+          <div>
+            {Object.entries(outputAndMatches).map(([output, match]) => (
+              <div key={output}>
+                <span className='white'>{output}</span>
+                <span className='orange'> = </span>
+                <span className='blue'>{match}</span>
+              </div>
+            ))}
+            <br/>
+            <span className='orange'>print</span><span className='white'>(num_of_antennas </span><span className='orange'>&gt; </span><span className='blue'>3 </span><span className='orange'>or </span><span className='white'>has_arms)</span><br/>
+            <span className='orange'>print</span><span className='white'>(</span><span className='orange'>not </span><span className='white'>has_ears)</span><br/>
+            <span className='orange'>print</span><span className='white'>(color </span><span className='orange'>!= </span><span className='blue'>&quot;red&quot;</span><span className='white'>)</span><br/>
+          </div>
+        </div>
```

Comment:
since we want this to be reusable, we could change to something like
create a `typings.ts` file with this
```
  interface bitOfCode {
    [key: string]: Color;
  }
  type lineOfCode = bitOfCode[];
```
something like this for input 
```
const code: lineOfCode[] = [
    [
      { ["num_of_antennas"]: Color.White },
      { [" = "]: Color.Orange },
      { ["2"]: Color.Blue },
    ],
    [
      { ["has_arms"]: Color.White },
      { [" = "]: Color.Orange },
      { ["True"]: Color.Blue },
    ],
    [
      { ["has_ears"]: Color.White },
      { [" = "]: Color.Orange },
      { ["False"]: Color.Blue },
    ],
    [
      { ["color"]: Color.White },
      { [" = "]: Color.Orange },
      { ['"white"']: Color.Blue },
    ],
    [],
    [
      { ["print"]: Color.Orange },
      { ["(num_of_antennas"]: Color.White },
      { [" > "]: Color.Orange },
      { ["3"]: Color.Blue },
      { [" or "]: Color.Orange },
      { ["has_arms)"]: Color.White },
    ],
    ...
  ];
```
and something like this for the `JSX` part
```
<div id="code">
          {code.map((lineOfCode) => (
            <>
              {lineOfCode.map((bitOfCode) =>
                Object.entries(bitOfCode).map(([text, color]) => {
                  return <span className={color}>{text}</span>;
                })
              )}
              <br />
            </>
          ))}
        </div>
```

Then you can make the this part it's own component, which gets passed in linesOfCode so that this can be reusable in the future!

Reply:
Thus, this should likely be split into the `shared` folder or something